### DISCUSSION TITLE
recommendation for production

### DISCUSSION THREAD

**Discussion by karazonanas** [shyim/shopware-docker#134] [Q&A]
Hello all,

I would like to know, if you would recommend this Shopware-Docker repository to use in production environment.

And I have a question:  in `$HOME/.config/swdc/env` there's `DEFAULT_DOMAIN` where I can define the sub part of the domain.
how can I use multiple projects which could have more than one sub part. Is it possible to define that for every project itself?

**Comment by shyim** [comment]
Hey,

I would not recommend using that on production. That https://github.com/shyim/shopware fits more for production.

Each folder in Code folder (default: ~/Code), gets a subdomain of the chosen `DEFAULT_DOMAIN`. Like when the folder is named `test` and the domain `dev.localhost` the resulting url will be `test.dev.localhost`. You can reconfigure the automaticly generated url using https://github.com/shyim/shopware-docker/wiki/Configuration#custom-configuration-for-virtual-hosts

**Reply by karazonanas** [reply]
thanks for your answer

the shopware container in https://github.com/shyim/shopware is reserving port 80 directly (without using bridge network). That would make it maybe difficult to host multiple shopware containers on the same server. Is there any solution for that?

**Reply by shyim** [reply]
You usually use something like https://doc.traefik.io/traefik/ before it. It obtains also ssl certificates and so on
